src/flamenco/runtime/fd_executor.c

```diff
@@ -919,15 +922,19 @@ fd_execute_instr( fd_exec_txn_ctx_t * txn_ctx,
       /* Log success */
       fd_log_collector_program_success( ctx );
     } else {
-      if( !txn_ctx->exec_err ) {
-        FD_TXN_ERR_FOR_LOG_INSTR( txn_ctx, exec_result, txn_ctx->instr_err_idx );
```

Comment:
Why removing this? This should add the error for all native programs instead of doing it for every program.